spec.html

```diff
@@ -7598,9 +7598,14 @@ <h1>Jobs and Host Operations to Enqueue Jobs</h1>
     </emu-note>
 
     <emu-clause id="sec-hostenqueuepromisejob" aoid="HostEnqueuePromiseJob">
-      <h1>HostEnqueuePromiseJob ( _job_ )</h1>
+      <h1>HostEnqueuePromiseJob ( _job_, _realm_ )</h1>
 
       <p>HostEnqueuePromiseJob is a host-defined abstract operation that schedules the Job abstract closure _job_ to be performed, at some future time. The abstract closures used with this algorithm are intended to be related to the handling of Promises, or otherwise, to be scheduled with equal priority to Promise handling operations.</p>
+      <p>The _realm_ parameter is passed through to hosts with no normative requirements; it is either *null* or a Realm.</p>
+
+      <emu-note>
+        The _realm_ for PromiseResolveThenableJobs is the result of calling GetFunctionRealm on the _then_ function object. The _realm_ for PromiseReactionJobs is the result of calling GetFunctionRealm on the handler if the handler is not *undefined*. Otherwise the _realm_ is *null*. The HTML spec, for example, uses _realm_ to check for ability to run script and to prepare to run script.
```

Comment:
```suggestion
        The _realm_ for PromiseResolveThenableJobs is the result of calling GetFunctionRealm on the _then_ function object. The _realm_ for PromiseReactionJobs is the result of calling GetFunctionRealm on the handler if the handler is not *undefined*. Otherwise the _realm_ is *null*. The WHATWG HTML specification, for example, uses _realm_ to check for ability to run script and to prepare to run script.
```

Reply:
Done